### PR TITLE
Add many folders and put files on them

### LINKED ISSUE #4: Tidy up repo
Tidy up repository by adding folders and putting files on those folders.

### PR DESCRIPTION
**Pull request by Catarina-Alves**
Re-organize the repo by adding folders and distributing files within them.

### CODE REVIEW

**Review by glukhove — COMMENTED** [review]
Will we have a conflict in README file? We have another pull request from @masha-astro.
And it's quite difficult to review when there are 1252 files :)
Do we need all of them? May be just ~10 pictures as examples?

File: data/download_data_scripts/README.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Note on downloading data
+
+There are many ways to download/generate data. 
+Besides the scripts on this folder, similar data can de generated using the notebooks to load data on `example_notebooks`.
```
Comment (glukhove) [inline data/download_data_scripts/README.txt:4]: Do you think we need to include changes to README into this pull request? @masha-astro  created another one specifically for README file. I just want to make sure that there will not be a conflict.

**Review by glukhove — COMMENTED** [review]
Just to check README. If changes are not critical, we can exclude them from current pull request and add them to another one. But it's up to you.

File: data/download_data_scripts/README.txt (hunk 1 above)
Comment (Catarina-Alves) [reply data/download_data_scripts/README.txt:4]: This is a different `README`. It is on the folder `data/download_data_scripts` so there will be no conflicts here. It is just extra information for people using that specific folder

Comment (glukhove) [reply data/download_data_scripts/README.txt:4]: Ah, sorry.
I'm good then.

**Comment by Catarina-Alves** [comment]
Solves #4

**Review by glukhove — APPROVED** [review]
Everything looks nice

**Pull request merged**